app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/action/attribute.phtml

```diff
@@ -11,11 +11,12 @@
 <form action="<?= /* @escapeNotVerified */ $block->getSaveUrl() ?>" method="post" id="attributes-edit-form" class="attributes-edit-form" enctype="multipart/form-data">
     <?= $block->getBlockHtml('formkey') ?>
 </form>
-<script>
-require(['jquery', "mage/mage"], function(jQuery){
-
-    jQuery('#attributes-edit-form').mage('form')
-        .mage('validation', {validationUrl: '<?= /* @escapeNotVerified */ $block->getValidationUrl() ?>'});
-
-});
+<script type="text/x-magento-init">
+    {
+        "#attributes-edit-form": {
+            "Magento_Ui/catalog/product/edit/attribute": {
```

Comment:
File catalog/product/edit/attribute.js located in Magento_Catalog module instead of Magento_Ui